src/device/pf_cmrpc.c

```diff
@@ -799,6 +799,33 @@ void pf_set_error (
    p_stat->pnio_status.error_code_2 = code_2;
 }
 
+void pf_set_error_if_not_already_set (
+   pnet_result_t * p_stat,
+   uint8_t code,
+   uint8_t decode,
+   uint8_t code_1,
+   uint8_t code_2)
+{
+   if (
+      p_stat->pnio_status.error_code == PNET_ERROR_CODE_NOERROR &&
+      p_stat->pnio_status.error_decode == PNET_ERROR_DECODE_NOERROR)
+   {
+      LOG_INFO (
+         PNET_LOG,
+         "CMRPC(%d) Setting error: 0x%02X 0x%02X 0x%02X 0x%02X (not "
+         "overwriting)\n",
+         __LINE__,
+         (unsigned)code,
+         (unsigned)decode,
+         (unsigned)code_1,
+         (unsigned)code_2);
+      p_stat->pnio_status.error_code = code;
+      p_stat->pnio_status.error_decode = decode;
+      p_stat->pnio_status.error_code_1 = code_1;
+      p_stat->pnio_status.error_code_2 = code_2;
+   }
```

Comment:
Add log of current error as well as ignored error code?

Reply:
Done